src/GenericExtractor.php

```diff
@@ -89,6 +89,9 @@ protected function createClient(Config $config): RestClient
             'proxy' => $this->proxy,
             // http://docs.guzzlephp.org/en/stable/request-options.html#verify-option
             'verify' => $this->api->hasCaCertificate() ? $this->api->getCaCertificateFile() : true,
+            // timeouts
+            'connect_timeout' => $this->api->getConnectTimeout(),
+            'timeout' => $this->api->getRequestTimeout(),
```

Comment:
Zistilo sa, ze niektorym klientom nestaci `30s` timeout, a potrebuju si ho nastavit.
Myslel som, ze `api.http.defaultOptions` sa daju na to pouzit, ale nedaju, maju iny zmysel.
Tak som pridal 2 nove polozky, a dam to aj do dokumentacie.